lib/java/src/org/apache/thrift/server/THsHaServer.java

```diff
@@ -154,7 +154,10 @@ protected static ExecutorService createInvokerPool(Args options) {
 
     return invoker;
   }
-
+  
+  protected ExecutorService getInvoker() {
```

Comment:
This could probably be static, given the method above is also static?  Thoughts?

Reply:
Yes, you are right. I will close this PR, as I'm unable to add commit to this PR (remove remote repo) and create a new one.
Thank you.